src/TokenParser/ApplyTokenParser.php

```diff
@@ -32,11 +32,7 @@ final class ApplyTokenParser extends AbstractTokenParser
     public function parse(Token $token): Node
     {
         $lineno = $token->getLine();
-        $name = $this->parser->getVarName();
-
-        $ref = new TempNameExpression($name, $lineno);
-        $ref->setAttribute('always_defined', true);
```

Comment:
was this unnecessary ?

Reply:
Because TempNameExpression does not support this attribute anyway.